sub_circuit_identification/src/write_verilog_lef.py

```diff
@@ -323,7 +323,9 @@ def generate_lef(fp, name, values, available_block_lef,
                      " -b " + block_name +
                      " -n " + str(unit_size_mos) +
                      " -X " + xval +
-                     " -Y " + yval )
+                     " -Y " + yval +
+                     " --width " + str(values['w']) +
+                     " --length " + str(values['l']))
```

Comment:
@kkunal1408 I am making these changes to work only for finfet technologies. I believe some additional work is needed for non-finfet technologies where no_units is determined by values['l'].

Reply:
It's fine for now, we can fix them later.